### ISSUE TITLE
Events page doesn't auto update

### ISSUE THREAD

**Issue by Sheldonwl** [aptakube/aptakube#46]
### Current and expected behavior

In K9s the events page auto refreshes and shows all new events and that does not seem to happen in the app. It also seems to be behind by quite a bit, see comparison images between k9s and Aptakube for the events in the monitoring namespace:

<img width="1768" alt="image" src="https://user-images.githubusercontent.com/6817520/212538622-ac9c66b8-34de-40d1-b62f-7458f9dc76f3.png">
<img width="1213" alt="image" src="https://user-images.githubusercontent.com/6817520/212538645-64af8c04-5c2a-4007-96b5-5b2e846d46c2.png">

### Operating System

MacOS 12.6

### Environment

clientVersion:
  buildDate: "2022-08-17T18:54:23Z"
  compiler: gc
  gitCommit: 95ee5ab382d64cfe6c28967f36b53970b8374491
  gitTreeState: clean
  gitVersion: v1.24.4
  goVersion: go1.18.5
  major: "1"
  minor: "24"
  platform: darwin/amd64
kustomizeVersion: v4.5.4
serverVersion:
  buildDate: "2022-07-13T14:19:57Z"
  compiler: gc
  gitCommit: c1de2d70269039fe55efb98e737d9a29f9155246
  gitTreeState: clean
  gitVersion: v1.23.9
  goVersion: go1.17.11
  major: "1"
  minor: "23"
  platform: linux/amd64

**Comment by goenning** [comment]
This is an odd one, the events table uses the same list+watch mechanism as the other tables.

Is everything ok with the other resources? Do they align with what k9s is showing and is it being updated at the same rate as k9s or is it lagging behind?

**Comment by goenning** [comment]
I've been trying to replica this, but unsuccessfully. I thought it could be different namespaces, but it seems like you already confirmed they are both looking at the same NS

This is what I have:
![image](https://user-images.githubusercontent.com/94755/212558485-ac013453-b0f1-45e1-bedd-333d8d1a9ef1.png)

**Comment by Sheldonwl** [comment]
Hmm, then it might have just been on my side. Perhaps too many tools trying to reach the k8s api at once. I noticed the other list updated as fast or even just a bit faster than k9s, so I also found it strange the the events weren't in sync. 

I've tested on another cluster, and it seems just fine at the moment. Don't expect that to change either, if it's using the same mechanism, but I'll test a bit more in the coming week and will let you know.
